Documentation/network/lb-ipam.rst

```diff
@@ -50,11 +53,26 @@ After adding the pool to the cluster, it appears like so.
     NAME        DISABLED   CONFLICTING   IPS AVAILABLE   AGE
     blue-pool   false      False         65788           2s
 
-.. note::
-    The amount of available IPs in the pool is lower than the actual sum of all
-    usable IPs in the CIDRs because the allocation logic is limited to 65536 IPs
-    per CIDR. CIDRs containing more than 65536 IPs can be broken down into multiple
-    smaller CIDRs to achieve full utilization.
+CIDRs, Ranges and dodgy IPs
+---------------------------
+
+An IP pool can have multiple blocks of IPs. A block can be specified with CIDR
+notation (<prefix>/<bits>) or a range notation with a start and stop IP. As
+pictured above.
+
+CIDRs are often used to specify routable IP ranges. By convention, the first
+IP of a range and the last IP of a range are reserved. The first IP is the 
+"network address" and the last IP is the "broadcast address". In most networks
+these IPs are not usable. In LB-IPAM we refer to these as "Dodgy IPs" because
+they do not always play well with all network equipment. LB-IPAM will not assign
+these by default. Exceptions are /32 and /31 IPv4 ranges and /128 and /127 IPv6
+ranges since these only have 1 or 2 IPs respectively.
+
+If you wish to use these dodgy IPs, you can set the ``.spec.allowDodgyIPs`` field
+to ``true``.
+
+Since ranges are typically used to indicate sub-sections of routable IP ranges,
```

Comment:
```suggestion
Since Ranges are typically used to indicate subsections of routable IP ranges,
```
I would use an uppercase "Range" here to be consistent with the header and make it distinct from a more generic meaning of an IP range (that can be also a CIDR).